ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -412,6 +413,16 @@ private void printAsTable(KsqlEntity ksqlEntity) {
     printTable(columnHeaders, rowValues, header, footer);
   }
 
+  private static String getTopicReplicaInfo(List<Integer> replicaSizes) {
+    if (replicaSizes.isEmpty()) {
+      return "0";
+    } else if (replicaSizes.stream().distinct().limit(2).count() <= 1) {
```

Comment:
it seems to me that the purpose of this check is to print a 'summary' if the replica sizes of all partitions is the same. So we would just print '3'  instead of '3, 3, 3'. Would be good to add a comment to this effect: will save mental effort later.

Reply:
or extract it into a method with a good name so we don't need comments that go out of date